### PR TITLE
fix: mobile ui/ux responsive adjustments

### LINKED ISSUE #184: Mobile UI/UX Responsive Adjustments
Make basic adjusment and fixes to make usable,

### PR DESCRIPTION
**Pull request by TopETH**
Close #184 

## Summary by CodeRabbit

- **Style Updates**
  - Updated various container class names from `"container"` to `"fluid-container"` for improved layout flexibility.
  - Modified the `SearchInput` component's styling and placeholder text for better user experience.
  - Adjusted the `UserMenu` to display the correct username and email from the user object.

- **New Features**
  - Enhanced `ThreadAccordion` component to conditionally remove the `threadId` query parameter.
  - Simplified height calculation in the `Sidebar` component for a more consistent layout.

- **Bug Fixes**
  - Corrected a typo in a console log message from `'Thereads'` to `'Threads'`.

- **Refactor**
  - Updated the `ChatLayout` structure to include additional wrapping `<div>` elements for better layout control.
  - Improved conditional rendering in the `ThreadHeading` component based on the `isBot` variable.

- **Chores**
  - Adjusted CSS in `globals.css` for better styling consistency across components.

### CODE REVIEW

File: apps/masterbots.ai/components/routes/c/sidebar/sidebar-responsive.tsx [hunk 1]
```diff
@@ -9,7 +9,8 @@ export async function ResponsiveSidebar() {
     transition-all
     -translate-x-full duration-500 ease-in-out
     data-[state=open]:translate-x-0 data-[state=closed]:lg:translate-x-0
-    w-[300px] lg:w-[250px] xl:w-[300px]"
+    w-[300px] lg:w-[250px] xl:w-[300px]
+    lg:static absolute"
```
[the remark this answers is not in the document]
Comment (AndlerRL) [reply apps/masterbots.ai/components/routes/c/sidebar/sidebar-responsive.tsx:13]: @TopETH -- Maybe it is not related, but can you check very small device screens? I noticed that when we have a device with less than 400px of width is making it look this way on Chat:

[Screencast from 05-13-2024 12:26:54 PM.webm](https://github.com/bitcashorg/masterbots/assets/35474730/a30c768c-a109-47da-b55d-160f2c871586)

**Review by AndlerRL — CHANGES_REQUESTED** [review]
Changes LGTM! I found while testing though that the sidebar and chat in general layout is not rendering good on small mobile devices. Can you check that comment? After that it will be ready to merge.

**Comment by TopETH** [comment]
> Changes LGTM! I found while testing though that the sidebar and chat in general layout is not rendering good on small mobile devices. Can you check that comment? After that it will be ready to merge.

@AndlerRL Could you please share the screenshot? Thanks.

**Review by AndlerRL — APPROVED** [review]
LGTM! Thanks for taking care on the details. Merging!

**Comment by AndlerRL** [comment]
@gaboesquivel -- This will be the last PR merged to `alpha` branch. After that I will block this branch to not receive more PRs after having the latest work on UI/UX to `search` branch. 

To do the merge with the latest on search, what I will be doing is the plan shared the other day but instead, I am going to include the database seeds on the merge to `search` branch.

cc - @merivercap @TopETH @leandrogavidia @Bran18 @AnoukRImola

**Comment by AndlerRL** [comment]
[Review and Fix Alpha UI/UX Regressions](https://trello.com/c/9tuJtZmO/142-review-and-fix-alpha-ui-ux-regressions)

**Pull request merged**